lib/smart_answer_flows/calculate-statutory-sick-pay/questions/coronavirus_related.govspeak.erb

```diff
@@ -0,0 +1,8 @@
+<% content_for :title do %>
```

Comment:
Do we need to add the content for the `coronavirus_self_or_cohabitant?` question as well?

Reply:
Oops, missed a couple of `git add`s.